common/src/main/java/com/wynntils/gui/screens/settings/elements/CustomColorConfigOptionElement.java

```diff
@@ -25,7 +29,7 @@ public void renderConfigAppropriateButton(
 
         CustomColor value = (CustomColor) configHolder.getValue();
 
-        RenderUtils.drawRect(poseStack, value, 105, 0, 0, renderHeight, renderHeight);
+        RenderUtils.drawRect(poseStack, value, 85, 0, 0, renderHeight, renderHeight);
```

Comment:
Qodana says:

 Suspicious variable/parameter name combination

'renderHeight' should probably not be passed as parameter 'width'

What do you have to say to your defense? :)

Reply:
Square.

Reply:
A square and fair defence! :)